anker/tests/tests/sell_rewards.rs

```diff
@@ -13,6 +13,7 @@ async fn test_successful_sell_rewards() {
     context
         .initialize_token_pool_and_deposit(Lamports(DEPOSIT_AMOUNT))
         .await;
+    context.fetch_pool_price().await;
```

Comment:
Hah, so this is a good test for the bad behavior now, we should keep a copy of this test but expect it to fail. You shouldn’t be able to successfully `sell_rewards()` after calling `fetch_pool_price()` just once, it needs to be called 5 times, the point was that an attacker would need to sandwich at least 3 of those 5 calls, which is expensive, and also less likely to succeed than sandwiching just one.